test/stripe/ephemeral_key_test.rb

```diff
@@ -0,0 +1,103 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class EphemeralKeyTest < Test::Unit::TestCase
+    FIXTURE = {
+      'id': 'ephkey_123',
+      'object': 'ephemeral_key',
+    }
```

Comment:
We should try to get a pay-server generated fixture in for this before we merge this PR for consistency and accuracy. I can help with that once we flip those API endpoints to `stable`.